### ISSUE TITLE
Provide a license file

### ISSUE THREAD

**Issue by cihadturhan** [Instabug/Instabug-Flutter#128]
After running pod install, I see this warning
```

[!] A license was specified in podspec `instabug_flutter` but the file does not exist - /Users/username/Desktop/github/project_root/ios/.symlinks/plugins/instabug_flutter/LICENSE

[!] Unable to read the license file `../LICENSE` for the spec `instabug_flutter (0.0.1)`

[!] Unable to read the license file `../LICENSE` for the spec `instabug_flutter (0.0.1)`

```

**Comment by AliAbdelfattah** [comment]
Hi @cihadturhan, thanks for reporting this. We're looking into it.

**Comment by AliAbdelfattah** [comment]
Fixed on v9.1.6
